content/english/python-pixel/B.conditions&loops/A.If-else-condition/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "If-else Condition"
```

Comment:
This file should also be moved to the Python basics section, like the nested-loops file.